app/uk/gov/hmrc/tariffclassificationfrontend/controllers/AssignCaseController.scala

```diff
@@ -45,11 +45,11 @@ class AssignCaseController @Inject()(authenticatedAction: AuthenticatedAction,
     }
   }
 
-  def get(reference: String): Action[AnyContent] = authenticatedAction.async { implicit request =>
+  def get(reference: String): Action[AnyContent] = (verify.authenticate andThen verify.casePermissions(reference)).async { implicit request =>
```

Comment:
Can anyone do this? It doesn't have a permission

Reply:
good catch